### ISSUE TITLE
FSET Stopped working for me :(

### ISSUE THREAD

**Issue by 1775OoRah** [stackTom/FSEarthTiles#27]
Absolutely LOVE this tool! 2.08 stopped working during use one night(I must've pushed my computer to the max..), so I installed 2.09. It will not work correctly either. I keep getting these results that seem to go on infinitely...
![FSET is acting weird](https://github.com/stackTom/FSEarthTiles/assets/173103664/36cc4016-1fb3-484b-ba99-d0ab3be023ec)

And this:
![FSET is acting weird 2](https://github.com/stackTom/FSEarthTiles/assets/173103664/74fc7358-0c30-467c-a7de-eaff66d76bd2)

I'm at my wit's end guys! haha.. I even tried previous versions. I'm thinking that FSET is probably fine, but something in my PC is malfunctioning. I just don't know what..

**Comment by stackTom** [comment]
Glad you're enjoying it ;)

A couple questions to try to figure out what's going wrong:

1) Are you creating scenery for fs2004?

2) Also, which window gets stuck exactly? I don't see anything particularly wrong with the pictures you sent. Which window is the weird one?

**Comment by 1775OoRah** [comment]
- Yes sir, I'm creating for FS2004 - Your program has restored my faith in sticking with this sim for so long lol
- The top-left window is the one that opens up during the final stage. Lately, I've observed it creating lines of coding infinitely instead of blinking and closing automatically when the scenery is complete. While that window is running along, the window on the right shows bmps generating in my main FSET folder(never seen that happen)--also seeming to go on infinitely.
Unless Im going crazy, getting old, or both(probably) I dont remember these windows looking like this during the process. 

Thank you for your help and take care :)

**Comment by stackTom** [comment]
Thats all normal. The FS2004 SDK uses a program called resample.exe and imagetool.exe to create the final files that fs2004 reads as scenery. These programs take the images FSET supplies it and splits them into millions of smaller files. That's all perfectly normal and part of what it takes to create scenery for FS2004.

Both resample.exe and imagetool.exe are very slow, and there's nothing to do to speed them up. Imagetool.exe might even make your monitor dark or show corrupt images (why it does this I have no idea; but it does it when it's called outside of FSET as well).

Long story short, give it time. It's working as it should be. I suggest creating a small area and test to make sure you can see it in FS2004 (which I believe it will from what you're telling me). Bigger areas will just take some time because resample.exe and imagetool.exe are just inherently slow and quirky programs.

**Comment by 1775OoRah** [comment]
I just created a tiny scenery and FSET did finish the process. Yay! It did create a scenery folder with both scenery and texture subfolders. However, the scenery subfolder has tmf files, no bgls. The texture subfolder has .bmp files, but the TgaSourceFiles subfolder is empty. Dangit! lol 
Thank you for your responses thus far. I don't mean to take up too much of your time. I've been using FSET for a fews years and I'm hooked!

Cheers!

**Comment by stackTom** [comment]
Don't worry, I don't mind providing support. That all sounds correct. As long as the scenery displays in FS2004, then it's all good. The contents of TgaSourceFiles will be deleted based on `FS2004KeepTGAs` in your FSEarthTiles.ini.

**Comment by 1775OoRah** [comment]
AAAhhh ok that makes sense as far as the TGAs. Got it! Now, I should have a bgl file in the scenery folder once the process is complete right?
This is what I'm left with after FSET completely finishes.
![FSET is acting weird 5](https://github.com/stackTom/FSEarthTiles/assets/173103664/93f76956-0827-4e12-99f6-4f6139581183)

**Comment by stackTom** [comment]
Yes, that looks correct.

**Comment by 1775OoRah** [comment]
Sir, I'm confused lol Ok so you said the photo above looks correct--even though there is no .bgl in the folder? I even tried activating the scenery(just out of curiosity), but still no luck(presumably because there is no .bgl). I've got to figure out why I'm not getting a .bgl upon completion. I feel like its something simple I keep missing...

Take Care!

**Comment by stackTom** [comment]
Apologies. I am just now seeing your reply now. I thought your picture showed a BGL, but it just shows a TMF. Can you send your fsearthtiles.ini and fsearthmasks.ini so I can try it on my system? I'll help you figure out why it's not showing up in fs2004.

**Comment by 1775OoRah** [comment]
Greetings! So after starting completely over, I got FSET to work! After all is said and done, I was able to activate and view the scenery in FS2004. The only thing to verify with you now is, in my scenery sub-folder, there is a .BGL file and three .TMF files.. Again, after loading the scenery into the sim, everything looks to be correct including the autogen. So I suppose I should jist leave it alone? Take Care!

**Comment by stackTom** [comment]
Yes, if it's showing up in the sim then all is well👍.
